packages/ui/src/2_molecules/Tooltip/Tooltip.types.ts

```diff
@@ -0,0 +1,31 @@
+export enum TooltipPlacement {
```

Comment:
It seems like we haven't been 100% consistent with this but I'd suggest to use `camelCase` naming in `enums` (e.g. `bottomEnd, bottomStart, bottom, ...`). I was looking at our [Code Guidelines](https://sovryn.monday.com/docs/2733202752) and it says we should use `UpperCamelCase` but I believe that's for just `enum` names and not values.

 What do you think @soulBit ?

Reply:
@pietro-maximoff sorry I wasn't specific enough in my comment but as @soulBit mentioned, this is mainly about `keys` (`values` can stay as they are now) so change it pls (e.g. `BOTTOM_END` will be `bottomEnd`).